### PR TITLE
Remove WeasyPrint and Flask-WeasyPrint lib requirements

### PR DESCRIPTION
**Pull request by northwestwitch**
Since we have switched to PDFKit, remove the WeasyPrint dependency

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,19 +9,20 @@ path.py
...
 livereload
 python-dateutil
 pymongo>=3.7,<4.0
 pathlib
-WeasyPrint==0.42.3
 pdfkit
 xlsxwriter
 click
 cryptography<3.4
+defusedxml
```
Comment (northwestwitch) [inline requirements.txt:23]: These ones were probably part of Flask-WeasyPrint before

Comment (dnil) [reply requirements.txt:23]: Yes, requirements for it and WeasyPrint likely. Ok, good.

**Comment by northwestwitch** [comment]
I'll test installing this one on cg-vm1!

**Comment by northwestwitch** [comment]
It's on stage now. Looks like it's working fine

**Review by dnil — APPROVED** [review]

**Pull request merged**